python-lib/dku_deeplearning_image/utils.py

```diff
@@ -77,25 +78,39 @@ def can_use_gpu():
     return len(tf.config.experimental.list_physical_devices('GPU')) > 0
 
 
-def set_gpu_options(should_use_gpu, gpu_list, memory_limit):
+def set_gpu_options(should_use_gpu, gpu_list, gpu_memory_allocation_mode, memory_limit_ratio=None):
     log_info("load_gpu_options")
     if should_use_gpu and can_use_gpu():
         log_info("should use GPU")
         gpus = tf.config.experimental.list_physical_devices('GPU')
```

Comment:
Perhaps better to use the official (non-experimental) API? https://www.tensorflow.org/api_docs/python/tf/config/list_physical_devices

Reply:
True, however I use tf 2.2 here so, although this function is available in non experimental modes, it is not the case on all functions. Fixed in 36eb1ba

Reply:
Why not switch to tf 2.4?